ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -1111,7 +1111,7 @@ getModSummaryFromImports env fp _modTime mContents = do
     modLoc <- liftIO $ if mod == mAIN_NAME
         -- specially in tests it's common to have lots of nameless modules
         -- mkHomeModLocation will map them to the same hi/hie locations
-        then mkHomeModLocation dflags (pathToModuleName fp) fp
+        then mkHomeModLocation dflags mod fp
```

Comment:
This seems to be causing long-path issues on windows.
I am unsure how relevant this still is... Anybody an idea?